packages/spin/spin.0.5.0/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+synopsis:
+  "Project scaffolding tool and set of templates for Reason and OCaml"
+description: """
+Project scaffolding tool and set of templates for Reason and OCaml
+"""
+maintainer: ["Thibaut Mattio"]
+authors: ["Thibaut Mattio"]
+license: "MIT"
+homepage: "https://github.com/tmattio/spin"
+doc: "https://tmattio.github.io/spin"
+bug-reports: "https://github.com/tmattio/spin/issues"
+depends: [
+  "ocaml" {>= "4.06.0" & < "4.10.0"}
```

Comment:
I'm curious, why is this package not working on OCaml 4.10?

Reply:
The current version of Reason on Opam does not work with OCaml 4.10.
Once https://github.com/ocaml/opam-repository/pull/15959 is merged, I will be able to remove the constraint!

Reply:
If you want. Though for the record, opam packages usually avoids adding too many upper-bound constraints as that many back-and-forth between opam-repository and individual devs are not scalable. Our CI ensures that things don't break, we put the appropriate constraints when it does and ping the dev when necessary.

Reply:
Didn't know, thanks for the explanation, I'll remove the upper bound!

Reply:
#15959 has now been merged